.circleci/config.yml

```diff
@@ -0,0 +1,38 @@
+# -*- yaml -*-
+# Use the latest 2.1 version of CircleCI pipeline process engine. See: https://circleci.com/docs/2.0/configuration-reference
+version: 2.1
+orbs:
+  sixrs-gke: sixriversystems/sixrs-gke@2
+jobs:
+  build:
```

Comment:
this should be broken down into multiple jobs for preparing, testing, building and linting like our other projects to make it more modular.  This also helps if s step fails and we want to rerun from failed.

Reply:
that's not a part of this PR. I'm not sure how it got here. I mean CI setup existed before this PR.